internal/policy/container/has_required_labels.go

```diff
@@ -67,7 +81,8 @@ func (p *HasRequiredLabelsCheck) Metadata() check.Metadata {
 
 func (p *HasRequiredLabelsCheck) Help() check.HelpText {
 	return check.HelpText{
-		Message:    "Check Check HasRequiredLabel encountered an error. Please review the preflight.log file for more information.",
-		Suggestion: "Add the following labels to your Dockerfile or Containerfile: name, vendor, version, release, summary, description",
+		Message: "Check HasRequiredLabel encountered an error. Please review the preflight.log file for more information.",
+		Suggestion: "Add the following labels to your Dockerfile or Containerfile: name, vendor, version, release, summary, description" +
```

Comment:
`maintainer` is also a required label now if i am reading correctly ?

Reply:
Good catch this also needs to be updated.